R/backend-teradata.R

```diff
@@ -217,20 +217,3 @@ sql_table_analyze.Teradata <- function(con, table, ...) {
 }
 
 utils::globalVariables(c("ATAN2", "SUBSTR", "DECIMAL", "WEEKNUMBER_OF_YEAR", "SUM"))
-
-#' @export
-#' @rdname win_over
-win_rank_tdata <- function(f) {
```

Comment:
Was this exported in a released dbplyr?

Reply:
It was, as of 2.3.0 (added in https://github.com/tidyverse/dbplyr/commit/4109051ff697446bc5a280d65aa708e568047f57#diff-c4f086980e5692d16e3337acd7bfefe3a9315135688133ee4f24ea2944ccba3bR167). I'd guess it was primarily to enable `row_number()` for Teradata backend.

Open to any recommended patterns for deprecation!